packages/xchain-crypto/package.json

```diff
@@ -46,17 +46,17 @@
   "devDependencies": {
     "@types/bech32": "^1.1.2",
     "@types/bip39": "^3.0.0",
-    "@types/crypto-js": "^4.0.1",
-    "@types/hdkey": "^0.7.1",
-    "@types/node": "^14.0.13",
-    "@types/uuid": "^8.3.0"
+    "@types/crypto-js": "^4.0.2",
+    "@types/hdkey": "^2.0.0",
+    "@types/node": "^16.7.10",
```

Comment:
We still stick on `14.x` (see https://github.com/xchainjs/xchainjs-lib/blob/c0c68856a5d1ee1ea3eaae5ce4a105990ea6573c/.nvmrc#L1 or in GH actions)

Reply:
@gromxyz Could you do a quick fix for Node version in GH actions, pls? See #427